nextflow_schema.json

```diff
@@ -1,47 +1,41 @@
 {
-    "$schema": "http://json-schema.org/draft-07/schema",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
     "$id": "https://raw.githubusercontent.com/genomic-medicine-sweden/nallo/master/nextflow_schema.json",
     "title": "genomic-medicine-sweden/nallo pipeline parameters",
     "description": "Long-read variant calling pipeline",
     "type": "object",
-    "definitions": {
+    "$defs": {
         "workflow_skip_options": {
             "title": "Workflow skip options",
             "type": "object",
             "description": "Allows skipping certain parts of the pipeline",
             "default": "",
             "properties": {
-                "skip_aligned_read_qc": {
+                "skip_qc": {
                     "type": "boolean",
                     "fa_icon": "fas fa-fast-forward",
-                    "description": "Skip QC of aligned reads",
+                    "description": "Skip QC of reads",
                     "default": false
                 },
-                "skip_raw_read_qc": {
-                    "type": "boolean",
-                    "fa_icon": "fas fa-fast-forward",
-                    "description": "Skip QC of unaligned (raw) reads",
-                    "default": false
-                },
-                "skip_short_variant_calling": {
+                "skip_snv_calling": {
                     "type": "boolean",
                     "description": "Skip short variant calling",
                     "fa_icon": "fas fa-fast-forward",
                     "default": false
                 },
-                "skip_assembly_wf": {
+                "skip_genome_assembly": {
                     "type": "boolean",
                     "description": "Skip genome assembly and assembly variant calling",
                     "fa_icon": "fas fa-fast-forward",
                     "default": false
```

Comment:
`nf-core pipelines schema` complains about the default not having the same type in the schema and config. This seems to be more an nf-core issue but I think that `false` is actually wrong here.